include/verilated.h

```diff
@@ -2385,6 +2385,7 @@ static inline WDataOutP VL_SEL_WWII(int obits, int lbits, int, int, WDataOutP ow
 /// Return QData from double (numeric)
 // EMIT_RULE: VL_RTOIROUND_Q_D:  oclean=dirty; lclean==clean/real
 static inline QData VL_RTOIROUND_Q_D(int bits, double lhs) VL_PURE {
+    (void) bits;
```

Comment:
```suggestion
static inline QData VL_RTOIROUND_Q_D(int, double lhs) VL_PURE {
```

Thanks for your first pull!   This is usually how we fix these; if you can try this and confirm it also fixes your warning I'll push.

Reply:
Test proposed fix and it works fine, forced pushed the proposed change in PR